engine/src/main/java/com/datatorrent/stram/util/ConfigUtils.java

```diff
@@ -68,15 +68,30 @@ public static String getRMUsername(Configuration conf)
     return principal;
   }
 
-  public static String getSchemePrefix(YarnConfiguration conf)
+  public static boolean isSecure(Configuration conf)
```

Comment:
Change the name to isSSLEnabled or something like that, secure in hadoop context typically refers to kerberos security